isce2_topsapp/packaging.py

```diff
@@ -92,21 +90,22 @@ def get_gunw_id(reference_properties: list,
     track_num = int(reference_properties[0]['pathNumber'])
     track = f'{track_num:03}'
 
-    # dates; remove dashes
-    reference_date = reference_properties[0]['startTime'].split('T')[0]
-    reference_date = reference_date.replace('-', '')
-    secondary_date = secondary_properties[0]['startTime'].split('T')[0]
-    secondary_date = secondary_date.replace('-', '')
+    # Center Datetimes
+    ref_center_datetime = get_center_time(reference_properties)
+    sec_center_datetime = get_center_time(secondary_properties)
+
+    # Center Time sring
+    ref_center_time_str = ref_center_datetime.strftime('%H%M%S')
+
+    reference_date_str = ref_center_datetime.strftime('%Y%m%d')
+    secondary_date_str = sec_center_datetime.strftime('%Y%m%d')
 
     # date pair
-    date_pair = f'{reference_date}_{secondary_date}'
+    date_pair = f'{reference_date_str}_{secondary_date_str}'
```

Comment:
As long as this is what goes into the GUNW filename, then I think we're good!

Reply:
Yes - look at the test and make sure!

The center datetimes of reference and secondary are now used to generate the GUNW id.

Reply:
https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/blob/e9dccb7b4b14326dacf0540cdd333a6f40f1a327/isce2_topsapp/packaging.py#L119-L136

Reply:
looks good for the filename.